plugins/available/vagrant-as-root.plugin.bash

```diff
@@ -0,0 +1,4 @@
+# Simple function for vagrant ssh that immediately switches to root.
+function vsshr() {
+      vagrant ssh $1 -- -t 'sudo su -; /bin/bash'
```

Comment:
Also think about showing an error message if `$1` is not provided. Or does vagrant show a descriptive error message in that case?